parent/pom.xml

```diff
@@ -177,6 +179,16 @@
     <version.jna>5.14.0</version.jna>
     <version.jopt-simple>5.0.4</version.jopt-simple>
 
+    <!-- GraphQL dependencies -->
+    <version.graphql-kickstart>15.1.0</version.graphql-kickstart>
+    <version.graphql-kickstart.tools>13.1.1</version.graphql-kickstart.tools>
+    <version.graphql-java>21.4</version.graphql-java>
+    <version.graphql-java.scalars>21.0</version.graphql-java.scalars>
+    <version.graphql-java.dataloader>3.2.0</version.graphql-java.dataloader>
+
+    <version.javassist>3.29.0-GA</version.javassist>
+    <version.json>20231013</version.json>
```

Comment:
❓ Do you know why we have these different json dependencies?

Reply:
tasklist uses `org.json:json` in the code
Zeebe and Operate use `com.vaadin.external.google:android-json` as a test dependency

I explicitly excluded `com.vaadin.external.google:android-json` from Tasklist 
as far  `com.vaadin.external.google:android-json` is used only with Test scope, we are safe to keep `org.json:json`

According to this [stackoverflow](https://stackoverflow.com/questions/52980064/maven-spring-boot-found-multiple-occurrences-of-org-json-jsonobject-on-the-cl) topic

I
```
org.json works great, but has a license clause that some people don't like ("The Software shall be used for Good, not Evil."). So Vaadin wanted to use the library, but couldn't be sure they wouldn't use it for evil someday. Instead, they re-implemented the interface, published android-json and used it as a drop in replacement for org.json. Others began to use android-json as well so that they too would not be bound by the requirement of not using their software for evil.
```

Reply:
`com.vaadin.external.google:android-json` is actually used by Operate and [I asked some weeks ago whether we can get rid of it. ](https://camunda.slack.com/archives/C9B5270DA/p1712229874151989)

Wondering why jackson or something is not working in this case? What is the use case here?

Reply:
The usecase is the creation of json Strings, we can definitively use `jakarta.json` instead
Created this issue for this matter https://github.com/camunda/zeebe/issues/17660